pkg/services/contexthandler/contexthandler.go

```diff
@@ -23,12 +23,11 @@ import (
 	"github.com/grafana/grafana/pkg/web"
 )
 
-func ProvideService(cfg *setting.Cfg, tracer tracing.Tracer, features featuremgmt.FeatureToggles, authnService authn.Service,
+func ProvideService(cfg *setting.Cfg, tracer tracing.Tracer, authnService authn.ServiceAuthenticateOnly,
```

Comment:
@kalleep is there any reason we would want the whole interface here? Wondering if replacing with `authn.Client` or another such limited interface a possiblity?

I want to use the `contexthandler` in our standalone flavor of Grafana, which has a lot less setup. There isn't very much wiresets for that either and I only really need `ext_jwt` authenticator.

Reply:
oh see https://github.com/grafana/grafana/pull/90102#discussion_r1666382141